pyproject.toml

```diff
@@ -54,7 +54,16 @@ env = [
   "INST_DB_USER=user",
   "INST_DB_PWD=user",
   "INST_DB_HOST=localhost:5432",
-  "INST_DB_NAME=filing"
+  "INST_DB_NAME=filing",
```

Comment:
should've probably caught this in one of the earlier PRs, the `INST_` prefix was meant for the institutions database; at that time in the user-fi module, I was thinking we may need to interact with more than just one db, thus the prefix to differentiate; since we're pretty much in the one db for the app paradigm, we can probably drop the prefix; or change it to probably the `FILING_` prefix.

Reply:
Ha I thought it meant "instance" like DB instance lol.  Will update to drop the prefix

Reply:
Ok updated, and wrote the following stories to update in docker-deploy and helm chart files, respectively:
https://github.com/cfpb/sbl-project/issues/131
https://github.cfpb.gov/reg-tech/regtech-eks-bootstrap/issues/42